src/pages/commits/ActiveCommits/CommitChartLabel.test.tsx

```diff
@@ -0,0 +1,81 @@
+import userEvent from "@testing-library/user-event";
+import { render, act } from "test_utils/test-utils";
+import { CommitChartLabel } from "./CommitChartLabel";
+
+const RenderCommitChartLabel = (version) => (
+  <CommitChartLabel
+    githash={version.revision.substring(0, 5)}
+    createTime={version.createTime}
+    author={version.author}
+    message={version.message}
+  />
+);
+
+afterEach(() => {
+  jest.clearAllTimers();
+  jest.clearAllMocks();
+});
+describe("CommitChartLabel", () => {
+  test("Displays author, githash and createTime", () => {
+    const { queryByDataCy } = render(RenderCommitChartLabel(versionShort));
+    expect(queryByDataCy("commit-label")).toHaveTextContent(
+      "4137c 6/16/21 11:38 PMMohamed Khelif"
+    );
+  });
+
+  test(
+    "Displays shortened commit message and the 'more' button if commit message" +
+      "is too long",
+    async () => {
```

Comment:
do we get the same result after removing `async`. I don't think we need it here since we don't use `await` in the function body